libraries/AP_Scheduler/PerfInfo.cpp

```diff
@@ -94,3 +95,15 @@ uint32_t AP::PerfInfo::get_stddev_time() const
 {
     return sqrt((sigmasquared_time - (sigma_time*sigma_time)/loop_count) / loop_count);
 }
+
+void AP::PerfInfo::update_logging()
```

Comment:
maybe more report_logging ? or send_logging ? I don't know.... or stay as it is also good enough